### DISCUSSION TITLE
What are the names of the missing icons?

### DISCUSSION THREAD

**Discussion by peteruithoven** [Rafostar/clapper#427] [General]
I'm running elementary OS 7.1 Horus (Built on Ubuntu 22.04.3 LTS), installed Clapper as Flatpak and I seem to be missing 2 icons. I understand from [this comment](https://github.com/Rafostar/clapper/issues/196#issuecomment-1003363470) that this is not a Clapper but an icon theme issue. So I'd like to report the issue, but I'm quite sure I need the name of the missing icons.

The over amplified volume icon and all the speed icons seem to be missing.
Screenshots:
![Screenshot from 2024-04-24 23 54 32@2x](https://github.com/Rafostar/clapper/assets/523210/85542c6f-718c-42f6-8366-17679a508484)
![Screenshot from 2024-04-24 23 55 11@2x](https://github.com/Rafostar/clapper/assets/523210/994a22b5-ac82-4eca-8dc6-2aa24aa7a0a0)

Based on:
- https://github.com/Rafostar/clapper/blob/master/src/lib/clapper-gtk/clapper-gtk-utils.c#L89
- https://github.com/Rafostar/clapper/blob/master/src/lib/clapper-gtk/ui/clapper-gtk-extra-menu-button.ui#L116
- https://github.com/Rafostar/clapper/blob/master/src/lib/clapper-gtk/clapper-gtk-utils.c#L96-L99

I'm thinking [elementary OS icons](https://github.com/elementary/icons) is missing:
- `audio-volume-overamplified-symbolic`
- `power-profile-power-saver-symbolic`
- `power-profile-balanced-symbolic`
- `power-profile-performance-symbolic`

Does that make sense?

**Comment by peteruithoven** [comment]
Hm, and instead of an info icon I see a star, I'm assuming that's `help-about-symbolic`?
![Screenshot from 2024-04-25 01 01 06@2x](https://github.com/Rafostar/clapper/assets/523210/c25d36f4-291d-4d52-a66e-a2481b05cbae)
Based on:  
https://github.com/Rafostar/clapper/blob/master/src/bin/clapper-app/ui/clapper-app-headerbar.ui#L116

**Comment by Rafostar** [comment]
Yes, you are correct about all 5 names. I mainly test/target vanilla GNOME with its "Adwaita" icon theme. While I do not want to force people to use it too, I cannot guarantee that every theme has the same icons as part of it.

I have seen ideas for apps to ship the icons which they use to show as fallback, but (currently???) I decided to not do that. It would not fix your "star" problem and other icons would look out of place anyway.

**Comment by peteruithoven** [comment]
I've opened 2 issues:
- https://github.com/elementary/icons/issues/1265
- https://github.com/elementary/icons/issues/1266
